Annotation/constituent_golden_set.json

```diff
@@ -759,7 +875,7 @@
                     }
                 ],
                 "type": "constituent",
-                "clause": "",
+                "clause": "why councils are ploughing money into fracking when support for it is consistently low",

```

Comment:
This might be the complement of a noun (questions - i.e. X asks questions as to why...)

Reply:
ok, lets remove this one